tests/validation_suite/.dockerignore

```diff
@@ -0,0 +1,2 @@
+#Ignore documentation files
+*.md
```

Comment:
You have some .md files which will required for documentation on the framework. No need to ignore it.

Reply:
Removed .md ignore.